code/game/machinery/floodlight.dm

```diff
@@ -61,12 +61,12 @@
 
 	if(on)
 		on = 0
-		set_light(0)
+		kill_light()
 	else
 		if(!cell || !cell.charge > powercost)
 			return
 		on = 1
-		set_light(brightness_on)
+		set_light()
```

Comment:
arg?

Reply:
With no arg, it will just get the lights from the atom level.